app/src/components/DayOfWeek.js

```diff
@@ -1,72 +1,100 @@
 import * as React from "react";
 
+import Button from "react-bootstrap/Button";
 import Card from "react-bootstrap/Card";
 import ListGroup from "react-bootstrap/ListGroup";
 import ListGroupItem from "react-bootstrap/ListGroupItem";
+import { Link } from "react-router-dom";
 
 import { moonPhases } from "../MoonPhases";
 import * as apiClient from "../apiClient";
 
-const DayOfWeek = ({ date, svg, data, tasks }) => {
+const DayOfWeek = ({ date, svg, data, tasks, userId }) => {
   const [dayTasks, setDayTasks] = React.useState([]);
 
-  const loadDayTasks = async () =>
-    setDayTasks(await apiClient.getTasks(date.format("YYYY-MM-DD")));
+  const loadDayTasks = async () => {
+    setDayTasks(
+      await userId
+        .then((data) => data.id)
+        .then((id) => apiClient.getTasks(id, date.format("YYYY-MM-DD"))),
```

Comment:
The above 3 lines seem to be repeated in several components.  Maybe extract this into a helper function?